packages/components/src/Tooltip/Tooltip.css

```diff
@@ -0,0 +1,65 @@
+:root {
+  --tooltip--border-color: var(--color-lightBlue);
+  --tooltip--arrow-size: var(--space-small);
+  --tooltip--offset: calc(-1 * calc(var(--space-smaller) + 1px));
+}
+
+.shadowActivator {
```

Comment:
This is the best classname!

Reply:
good, that took a good few minutes to figure out haha